notebooks/tutorial/5 - Create tome.ipynb

```diff
@@ -96,8 +96,10 @@
    "outputs": [],
    "source": [
     "# Initialize our \"footsteps_by_rank\" tome\n",
+    "footsteps_tome_name = 'footsteps_by_rank.2022-05-15,2022-05-15'\n",
     "tomer = curator.make_tome(\n",
-    "    'footsteps_by_rank',\n",
+    "    footsteps_tome_name,\n",
+    "    header_tome_name=os.environ.get(\"PURESKILLGG_TOME_DEFAULT_HEADER_NAME\"),\n",
```

Comment:
Why did we need to add this line? Doesn't it use that env var automatically?

Reply:
bug introduced when i made the copier its own class. Fix is here, just gotta upgrade dsdk version. https://github.com/pureskillgg/dsdk/commit/99071007a9d8637f60eff9da1e1286c16f97fb85

Reply:
Fixed in latest DSDK